File: C/1005.c
#include <stdio.h>

int main()
{

    double n1, n2, media;
    scanf("%lf%lf", &n1, &n2);
    n1 = n1 * 3.5;
    n2 = n2 * 7.5;
    media = (n1 + n2) / 11.0;

    printf("MEDIA = %.5lf\n", media);

    return 0;
}

File: GO/1005.go
package main

import "fmt"

func Media(a, b float64) float64 {
	return (a*3.5 + b*7.5) / 11.0
}
func main() {
	var a, b float64

	fmt.Scan(&a)
	fmt.Scan(&b)

	fmt.Printf("MEDIA = %.5f\n", Media(a, b))
}
